tests/test_pages.py

```diff
@@ -32,6 +32,12 @@ def session_capabilities(session_capabilities):
     return session_capabilities
 
 
+@pytest.fixture
+def selenium(selenium):
+    selenium.set_window_size(2560, 1440)
```

Comment:
Not everyone has a 1440p screen. I'd start a bit safer with 1080p.
```suggestion
    selenium.set_window_size(1920, 1080)
```